### PR TITLE
Fixes #25731 - fix the setting description

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by ares** [comment]
@lhellebr mind to check the string now?

**Comment by lhellebr** [comment]
@ares Seems ok now

File: app/models/setting/provisioning.rb [hunk 1]
```diff
@@ -66,7 +66,7 @@ def self.default_settings
       self.set('intermediate_ipxe_script', N_('Intermediate iPXE script for unattended installations'), 'iPXE intermediate script', N_('iPXE intermediate script'), nil, { :collection => Proc.new { Hash[ProvisioningTemplate.unscoped.of_kind(:iPXE).map { |tmpl| [tmpl.name, tmpl.name] }] } }),
       self.set(
         'destroy_vm_on_host_delete',
-        N_("Destroy associated VM on host delete. When enabled, VMs linked to Hosts will be deleted on Compute Resource, meaning they can be re-associated or imported back to Foreman again. This does not automatically power off the VM"),
+        N_("Destroy associated VM on host delete. When enabled, VMs linked to Hosts will be deleted on Compute Resource. When disabled, VMs are unlinked first, meaning they remain on Compute Resource and can be re-associated or imported back to Foreman again. This does not automatically power off the VM"),
```
Comment (tbrisker) [inline app/models/setting/provisioning.rb]:
```suggestion
        N_("Destroy associated VM on host delete. When enabled, VMs linked to Hosts will be deleted on Compute Resource. When disabled, VMs are unlinked when the host is destroyed, meaning they remain on Compute Resource and can be re-associated or imported back to Foreman again. This does not automatically power off the VM"),
```

Comment (lhellebr) [reply app/models/setting/provisioning.rb]: s/destroyed/deleted/

Comment (ares) [reply app/models/setting/provisioning.rb]: updated including both suggestions

**Review by tbrisker — APPROVED** [review]
Thanks @ares !

**Pull request merged**